index.html

```diff
@@ -110,8 +110,12 @@ <h2 class="card-header">CC Ignite</h2>
     <section class="speakers">
       <h2>Featured Speakers</h2>
       <div class="speaker-container">
-        <div class="speaker-card">
-          <img src="./images/Itachi_Uchiha001.webp" alt="" />
+        <!-- <div class="speaker-card">
```

Comment:
Kindly, I suggest you remove the commented code, it's not a good practice, it will also make your code look cleaner and more professional.

Reply:
Done. Thank you for your reviews